### DISCUSSION TITLE
A Bit Of A Nitpick.

### DISCUSSION THREAD

**Discussion by Sam12543** [Rebirth-of-the-Night/Rebirth-Of-The-Night#748] [Q&A] [answered]
Okay so the glory of minecraft is that your builds don't get destroyed by really anything, its creativity, its something that by compiling this modpack you ruin. If i spend 10s of hours on building something I don't want it destroyed, when i spend 30 minutes on a building, I don't want it to be destroyed either. I'm new to the modpack so I really don't know the limitations to the zombies but they destroy 3 houses, a wall around a castle I build, and other small buildings, and it is not something that I appreciate. It took me hours to build these things and when they get destroyed countless times it is a pain in the ass and I would really appreciate anything to counter this. I know that it may not be an inconvenience to most players but it is seriously annoying and being that the mod authors probably aren't going to do anything about it, but until they stop doing this crap or I know a way to counter it, I'm going to stay away from this modpack for now. I would seriously like some insight on the un dead's capabilities, it would be really nice.

**Comment by democat3457** [comment, marked as answer]
This pack focuses more on conflict, combat, and exploration than building. Buildings and blocks are more geared towards functionality than vanilla's experience requires, so traps and such will be extremely useful early-game.

If you came here for building and building only, this is not your pack.
